### ISSUE TITLE
Angular 7 Support

### ISSUE THREAD

**Issue by mitschmidt** [rednez/angular-user-idle#25]
Could this be implemented? Probably a pure formality.

**Comment by zandroid** [comment]
I have tried it with Angular 7 and it works fine for me. @mitschmidt, @mtotheoneandonlyk please post more details about your problem or ideas why it should not work.

**Comment by mitschmidt** [comment]
While it does actually work, the problem ist that npm reports a version mismatch:

```
npm[app] WARN angular-user-idle@2.0.0 requires a peer of @angular/common@^6.0.0-rc.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.

npm[app] WARN angular-user-idle@2.0.0 requires a peer of @angular/core@^6.0.0-rc.0 || ^6.0.0 but none is installed.You must install peer dependencies yourself.
```

**Comment by rednez** [comment]
Will fix in 6.0.1.
